test/test_rohmu.py

```diff
@@ -0,0 +1,163 @@
+import hashlib
+import logging
+import os
+import pathlib
+from tempfile import NamedTemporaryFile
+
+import pytest
+from rohmu import get_transfer, rohmufile
+from rohmu.object_storage.base import BaseTransfer
+from rohmu.rohmufile import create_sink_pipeline
+from rohmu.typing import Metadata
+
+from .base import CONSTANT_TEST_RSA_PRIVATE_KEY, CONSTANT_TEST_RSA_PUBLIC_KEY
+
+EMPTY_FILE_SHA1 = "da39a3ee5e6b4b0d3255bfef95601890afd80709"
+
+log = logging.getLogger(__name__)
+
+
+@pytest.mark.parametrize("compress_algorithm", ["lzma", "snappy", "zstd"], ids=["test_lzma", "test_snappy", "test_zstd"])
```

Comment:
Technically, you don't need the id. When the values are `str`, they will be used as is to uniquely name the test (I think).
For the `int` below, it makes sense though, as it improves readability.

Reply:
Good comment. First `ids` argument will be removed.